Appsilon/mbaza
# Prepare template

### Description
Stage 1 of #241. This PR merges to `upgrade-erb` branch, which contains a freshly initialized template in the project root and a `legacy` directory with the old project. Changes in this PR:
1. Upgrade packages to newest versions (the template uses rather recent versions, but not necessarily the latest).
2. Relax Electron security settings (necessary as #84 is still not resolved).
3. Add special handling for asset file paths, so e.g. `file:/assets/icon.png` works both in development and production.
4. Adjust automatic styling settings (print width) to better match the legacy project and cleanup the code.

### How to test
1. Follow the instructions in the README to install dependencies and launch the project.
2. Review the diff to see if there is anything suspicious :wink:
